doc/manual/install-judgehost.rst

```diff
@@ -50,15 +50,31 @@ home directory::
   make judgehost
   sudo make install-judgehost
 
+For the next section we assume a machine with possibly hyperthreading and more
+than 3 CPUs. This can be checked with::
+
+  lscpu | grep "Thread(s) per core"
+
+having a value above 1 indicates hyperthreading or::
+
+  cat /sys/devices/system/cpu/smt/active
+
+a value of `1` or `on`. The target CPU for the next session should be in the range of::
```

Comment:
I suggest to replace "target CPU for the next session" with "target CPU core to restrict the judgedaemon to below" as "session" is unclear.

Reply:
session is a typo and should be section. But even when the typo was not there your wording is nicer.